debezium-core/src/main/java/io/debezium/relational/TableSchemaBuilder.java

```diff
@@ -75,9 +77,11 @@ public TableSchemaBuilder(ValueConverterProvider valueConverterProvider,
                               CustomConverterRegistry customConverterRegistry,
                               Schema sourceInfoSchema,
                               FieldNamer<Column> fieldNamer,
-                              boolean multiPartitionMode) {
+                              boolean multiPartitionMode,
+                              EventConvertingFailureHandlingMode eventConvertingFailureHandlingMode) {
         this(valueConverterProvider, null, schemaNameAdjuster,
-                customConverterRegistry, sourceInfoSchema, fieldNamer, multiPartitionMode);
+                customConverterRegistry, sourceInfoSchema, fieldNamer, multiPartitionMode,
+                eventConvertingFailureHandlingMode);
```

Comment:
@nicholas-fwang Let's do it using slightly different way. There should be a new constructor that will accept `EventConvertingFailureHandlingMode`. The existing constructor will just use the value that conforms to the default behaviour.
Then only `MySqlDatabaseSchema` and MySQL conncetor in generl will be affected and we can introduce the config behaviour into other connectors if/when needed. WDYT?